gsy_framework/data_classes.py

```diff
@@ -322,12 +322,12 @@ def __init__(self, id: str, creation_time: DateTime, offer_bid: Union[Offer, Bid
         self.id = str(id)
         self.creation_time = creation_time
         self.time_slot = time_slot  # market slot of creation
-        self.offer_bid = offer_bid
+        self.order = order
         self.seller = seller
         self.buyer = buyer
         self.residual = residual
         self.already_tracked = already_tracked
-        self.offer_bid_trade_info = offer_bid_trade_info
+        self.trade_orders_info = trade_orders_info
```

Comment:
```suggestion
        self.traded_order_info = traded_order_info
```
Just my suggestion, looks better